packages/components/Avatar/src/useAvatar.ts

```diff
@@ -80,8 +80,8 @@ export const useAvatar = (props: AvatarProps): AvatarInfo => {
   return {
     props: {
       accessible: accessible ?? true,
-      accessibilityLabel: accessibilityLabel || accessibilityText,
-      accessibilityRole: accessibilityRole ?? 'image',
+      accessibilityLabel: accessibilityText,
+      accessibilityRole: accessibilityRole ?? (name ? 'none' : 'image'),
```

Comment:
This is not a good change for win32. The Avatar will be treated as an image with a name prop on win32, especially since it's possible that someone can pass in a name and an image as props.

Reply:
ok , on Android we're trying to match native behaviours, so this would be required here with platform check.